src/task_scl_eff_pot_hab.py

```diff
@@ -86,14 +100,25 @@ def __init__(self, *args, **kwargs):
             ee.ImageCollection(self.inputs["hii"]["ee_path"])
         )
         self.historic_range = ee.Image(self.inputs["historic_range"]["ee_path"])
-        self.extirpated = ee.Image(self.inputs["extirpated"]["ee_path"])
-        self.elevation = ee.Image(self.inputs["elevation"]["ee_path"])
         self.ecoregions = ee.FeatureCollection(self.inputs["ecoregions"]["ee_path"])
         self.density = ee.FeatureCollection(self.inputs["density"]["ee_path"])
         self.water = ee.Image(self.inputs["water"]["ee_path"])
         self.set_aoi_from_ee(self.inputs["probability"]["ee_path"])
 
     def calc(self):
+        def distance_km_to_pixels(distance_km, image_resolution):
```

Comment:
I would move these methods into the class root and use class properties (like `self.scale`) as defaults, where possible.